lib/src/react_client/synthetic_event_wrappers.dart

```diff
@@ -4,153 +4,621 @@
+  ///
+  /// See <https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/pointerId>
   external num get pointerId;
+
+  /// The width (magnitude on the X axis), in CSS pixels, of the contact geometry of the pointer.
+  ///
+  /// See <https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/width>
   external num get width;
+
+  /// The height (magnitude on the Y axis), in CSS pixels, of the contact geometry of the pointer.
+  ///
+  /// See <https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/height>
   external num get height;
+
+  /// The normalized pressure of the pointer input in the range `0` to `1`, where `0` and `1` represent the minimum and maximum
+  /// pressure the hardware is capable of detecting, respectively.
+  ///
+  /// See <https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/pressure>
   external num get pressure;
+
+  /// The normalized tangential pressure of the pointer input (also known as barrel pressure or cylinder stress) in the
+  /// range `-1` to `1`, where `0` is the neutral position of the control.
+  ///
+  /// See <https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/tangentialPressure>
   external num get tangentialPressure;
+
+  /// The plane angle (in degrees, in the range of `-90` to `90`) between the Y–Z plane and the plane containing both the pointer
+  /// (e.g. pen stylus) axis and the Y axis.
+  ///
+  /// See <https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/tiltX>
   external num get tiltX;
+
+  /// The plane angle (in degrees, in the range of `-90` to `90`) between the X–Z plane and the plane containing both the
+  /// pointer (e.g. pen stylus) axis and the X axis.
+  ///
+  /// See <https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/tiltY>
   external num get tiltY;
+
+  /// The clockwise rotation of the pointer (e.g. pen stylus) around its major axis in degrees, with a value in the range
+  /// `0` to `359`.
+  ///
+  /// See <https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/twist>
   external num get twist;
+
+  /// Indicates the device type that caused the event (mouse, pen, touch, etc.)
+  ///
+  /// See <https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/pointerType>
   external String get pointerType;
+
+  /// Indicates if the pointer represents the primary pointer of this pointer type.
+  ///
+  /// See <https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent/isPrimary>
   external bool get isPrimary;
 }
 
+/// A [SyntheticEvent] wrapper that is specifically backed by a [TouchEvent].
+///
+/// See <https://developer.mozilla.org/en-US/docs/Web/API/TouchEvent>
 @JS()
-@anonymous
 class SyntheticTouchEvent extends SyntheticEvent {
+  /// [SyntheticTouchEvent]s cannot be manually instantiated.
+  ///
+  /// Use `createSyntheticTouchEvent` instead.
+  external factory SyntheticTouchEvent._();
+
+  /// A value indicating whether or not the alt key was down when the touch event was fired.
```

Comment:
#nit Same as above, but also "or not" is redundant, and can just be
```suggestion
  /// Whether the alt key was down when this event was fired.
```